dhcpv6/nclient6/client.go

```diff
@@ -456,7 +457,7 @@ func (c *Client) SendAndRead(ctx context.Context, dest *net.UDPAddr, msg *dhcpv6
 			case <-c.done:
 				return ErrNoResponse
 
-			case <-time.After(timeout):
+			case <-time.After(c.timeout):
```

Comment:
I think this doesn't quite work since `c.timeout` remains constant throughout all the iterations instead of doubling each iteration.

Reply:
Changed this so that the timeout doubles every iteration.